requirements_hook.txt

```diff
@@ -1,6 +1,7 @@
 attrs>=21.4.0
```

Comment:
Does it make sense to merge `requirements_hook.txt` and `requirements.txt`?

Reply:
I like it that we have minimal dependencies for people that want to install it.

Maybe with a pre-commit hook environment as shown in the comment above we can drop the dev requirements all together?
They just include pytype, mypy and black iirc.

Reply:
Then maybe just rename requirements_hook to requirements-dev?